service/jargon/defaults.yaml

```diff
@@ -1,91 +1,291 @@
+#Default jargon - comment out as necessary. 
+
+#Note that some of these are experimental abbreviations which don't have any real purpose and are simply there to play with
+ 
+#shortcuts
+ 
 1: I
 2: II
-aar: Ancillary Armor Repairer
+lrg: Large
+hvy: Heavy
+med: Medium
+sml: Small
+mic: Micro
+xl: X-Large
+l: Large
+h: Heavy
+m: Medium
+s: Small
+ 
+#language fixes (american/british differences)
+ 
+armour: armor
+neutraliser: neutralizer
+stabiliser: stabilizer
+energised: energized
+economiser: economizer
+ 
+#definitions
+ 
+cn: Caldari Navy
+rf: Republic Fleet
+in: Imperial Navy
+fn: Federation Navy
+ 
+#weapons
+ 
+ac: AutoCannon
+rt: Artillery
+arty: Artillery
+bs: Blaster
+rg: Railgun
+pl: Pulse Laser
+bl: Beam Laser
+ml: Missile Launcher
+lnb: Light Neutron Blaster
```

Comment:
These sorts of abbreviations feel unnecessary, but it's probably a matter of taste/convenience.

Reply:
I partly agree. There's a lot of abbreviations in here that I've never really seen before in context . All the two letter keywords also have me worried about unintended side effects, such as bringing back more results than the user is expecting, and getting bug reports on them.

That being said, I think we'll try them and see if anyone has an issue with them. :)